apps/judicial-system/api/src/app/modules/file/file.controller.ts

```diff
@@ -54,22 +54,30 @@ export class FileController {
     )
   }
 
-  @Get('caseFilesRecord/:policeCaseNumber')
+  @Get([
```

Comment:
I did not realize this was possible 😮 interesting

Reply:
I think we should use this more as it could eliminate some code duplications.